README.md

```diff
@@ -129,6 +131,8 @@ The data is returned as a list of objects (seen below).
     isVerified: Boolean,
     isOfficialArtist: Boolean,
     hasOwnerReplied: false,
+    isMember: Boolean, // Whether the user that made the comment is a paid member or not
+    memberIcon: String | null // URL of the member icon
```

Comment:
I imagine `memberIcon` would be something like:
```js
memberIcon: {
  url: "URL",
  height: 100,
  width: 100,
  otherMetadata: "whatever",
}
```
Actually if you look at how you fetch the URL
`customBadge.thumbnails` is an array of objects not strings
```suggestion
    memberIconURL: String | null // URL of the member icon
```